assets/models/system/automl-ner/spec.yaml

```diff
@@ -20,5 +20,6 @@ properties:
   languages: en
 tags:
   license: apache-2.0
+  SharedComputeCapacityEnabled: ''
```

Comment:
I guess we should remove for this one and automl-text-classification one?
@skasturi

Reply:
Swathi remove for above two models. We can merge post that

Reply:
Removed for tags for Automl two models

Reply:
@novaturient95  Please review